### PR TITLE
docs: fix import locale

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by chenxch — APPROVED** [review]
Older referencing methods might be better served by considering comments and adding version boundary instructions.

**Review by tolking — COMMENTED** [review]
IMO, we can add tips to let users know when to use `es` `lib` or `dist`

- module

```ts
import zhCn from 'element-plus/es/locale/lang/zh-cn'
```

- commonjs

```cjs
const zh = require('element-plus/lib/locale/lang/zh-cn')
```

WTYT

**Comment by btea** [comment]
Maybe we should use `import zhCn from 'element-plus/es/locale/lang/zh-cn'` uniformly in the code of this page, `commonjs` seems unnecessary. 🤔

File: docs/en-US/guide/i18n.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ for globally configuring locale and other settings.
   import { defineComponent } from 'vue'
   import { ElConfigProvider } from 'element-plus'
 
-  import zhCn from 'element-plus/dist/locale/zh-cn.mjs'
+  import zhCn from 'element-plus/lib/locale/lang/zh-cn'
```
Comment (tolking) [inline docs/en-US/guide/i18n.md:40]:
```suggestion
  import zhCn from 'element-plus/es/locale/lang/zh-cn'
```

So changing it to es here may be more appropriate

**Review by tolking — APPROVED** [review]

**Pull request merged**